dotcom-rendering/src/lib/labs.ts

```diff
@@ -19,3 +20,14 @@ export const getOphanComponents = ({
 		ophanComponentLink: `labs-logo-${componentName}`,
 	};
 };
+
+export const getLabsUrlSuffix = (editionId: EditionId): string => {
+	switch (editionId) {
+		case 'AU':
+			return '-australia';
+		case 'US':
+			return '-us';
+		default:
+			return '';
+	}
```

Comment:
Nice function, quick question, we have the `EditionId` type with `AU | UK..` etc
What si the purpose of the `-` suffix?

Reply:
Take it or leave it, but I think that a function that returned a complete URL or pathname would be slighly clearer:

```ts
const getLabsPathname = (editionId: EditionId): '/guardian-labs' | '/guardian-labs-us' | '/guardian-labs-australia' => { /* … */ }
// or alternatively
const getLabsUrl = (editionId: EditionId): URL => { /* … */ }
```